content/en/references/configuration.md

```diff
@@ -322,9 +323,10 @@ To learn more about these configuration options, see [DNS Server]({{< ref "dns-s
 
 | Variable             | Example Values | Description |
 |----------------------|----------------|-------------|
-| `ACTIVATE_PRO`       | 1 (default)    | Whether pro should be activated or not. This is set to true by default if using the `localstack/localstack-pro` container image. If set to `1`, LocalStack will fail to start if the license key activation did not work. If set to `0`, an attempt is made to start LocalStack without pro features.
-| `LOCALSTACK_AUTH_TOKEN` |                | AUTH_TOKEN to activate LocalStack Pro.
-| `LOG_LICENSE_ISSUES` | 1 (default)    | Whether to log issues with the license activation to the console.
+| `ACTIVATE_PRO`       | `0` \| `1`&nbsp;(default)    | Whether Pro should be activated or not. This is set to true by default if using the `localstack/localstack-pro` container image. If set to `1`, LocalStack will fail to start if the license key activation did not work. If set to `0`, an attempt is made to start LocalStack without Pro features. |
```

Comment:
I take it the `&nbsp;` is to stop line wrapping?

Reply:
yes because having the (default) separated on a new line is awkward.

it's the `~` latex equivalent (for the academics)